### PR TITLE
30881 Feat: Move gordo client code

### PR DESCRIPTION
**Pull request by rkicz**
Copy code from gordo.client module, with small modifications.

-     gordo/client/client.py
-     gordo/client/dataframe.py
-     gordo/client/forwarders.py
-     gordo/client/io.py
-     gordo/client/utils.py

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -10,7 +10,9 @@ exclude =
     build,
     dist,
     venv,
-    tests
+    tests,
+    # TODO: temporarily disable flake8. Rafactor code.
```
Comment (o-fedorov) [inline setup.cfg:14]: Or use flakehell like we have for Latigo.

**Comment by o-fedorov** [comment]
Could you please note which code is copied-and-pasted from Gordo and which is added/modified?

File: gordo/client/__init__.py [hunk 2]
```diff
@@ -1 +1,6 @@
-__version__ = "0.1.0"
+from pkgutil import extend_path
+
+from gordo.client.client import Client
+from gordo.client.utils import influx_client_from_uri
+
+__path__ = extend_path(__path__, __name__)
```
Comment (o-fedorov) [inline gordo/client/__init__.py:6]: Is it really required? I was sure [PEP-420](https://www.python.org/dev/peps/pep-0420/)-compliant package structure is enough.

Comment (rkicz) [reply gordo/client/__init__.py:6]: yes, without it i cant import gordo.client from gordo

File: pyproject.toml [hunk 3]
```diff
@@ -23,6 +23,17 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
+pandas = "~=1.0"
+wrapt = "~=1.11"
+cachetools = "~=4.0"
+scikit-learn = "~=0.23"
+gordo-dataset = "~=2.3.0"
+tensorflow-estimator = "~=2.1.0"
```
[the remark this answers is not in the document]
Comment (rkicz) [reply pyproject.toml:31]: look's like no

File: tests/test_client.py [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+def _mock_response(mocked_responses, url, response_name):
+    client_response = gordo_responses[response_name]
+    mocked_responses.add(
+        method=client_response.method,
+        url=f"https://localhost:443{url}",
+        body=client_response.json if client_response.content_type == "application/json" else client_response.body,
+        status=client_response.status,
+        content_type=client_response.content_type,
+    )
+    return client_response
```
[the remark this answers is not in the document]
Comment (rkicz) [reply tests/test_client.py:57]: at some point it was like you wrote, but i think passing response_name is better - response readed once for all tests at the beginning of script and it's cleaner

File: gordo/client/__init__.py (hunk 2 above)
Comment (koropets) [reply gordo/client/__init__.py:6]: looks weird. at least it deserves a comment with an explanation of why we did this

Comment (rkicz) [reply gordo/client/__init__.py:8]: done

**Review by koropets — APPROVED** [review]

**Pull request merged**